crates/holochain/benches/multi_install.rs

```diff
@@ -0,0 +1,426 @@
+        network.tuning_params = Arc::new(tuning);
+        network.bootstrap_service = Some(url2::Url2::parse(url));
+        // network.bootstrap_service = Some(url2::url2!("https://bootstrap-staging.holo.host"));
+        network.transport_pool = vec![kitsune_p2p::TransportConfig::Quic {
+            bind_to: None,
+            override_host: None,
+            override_port: None,
+        }];
+        ConductorConfig {
+            network: Some(network),
+            admin_interfaces: Some(vec![AdminInterfaceConfig {
+                driver: InterfaceDriver::Websocket { port: 0 },
+            }]),
+            ..Default::default()
+        }
+    };
+    let mut configs = Vec::new();
+    for _ in 0..num_conductors {
+        configs.push(config());
+    }
+    let conductors = SweetConductorBatch::from_configs(configs.clone()).await;
+
+    let conductors = conductors.into_inner().into_iter();
+
+    let (dna, _) = SweetDnaFile::unique_from_test_wasms(vec![
+        holochain_wasm_test_utils::TestWasm::ValidateValid,
+    ])
+    .await
+    .unwrap();
+
+    let unit_entry_def = holochain_zome_types::EntryDef::default_with_id("unit");
+    let inline_zome = holochain_zome_types::InlineZome::new_unique(vec![unit_entry_def.clone()])
+        .callback(
+            "validate",
+            |_api, _data: holochain_zome_types::ValidateData| {
+                Ok(holochain::core::ribosome::guest_callback::validate::ValidateResult::Valid)
+            },
+        );
+    let (inline_dna, _) = SweetDnaFile::unique_from_inline_zome("app", inline_zome)
+        .await
+        .unwrap();
+    Producers {
+        conductors: conductors.collect(),
+        i: 0,
+        total: 0,
+        test_dna: dna,
+        inline_dna,
+        num_installs,
+        happ_path,
+    }
+}
+
+pub fn rt() -> Runtime {
+    Builder::new_multi_thread()
+        .enable_all()
+        .max_blocking_threads(24)
+        .build()
+        .unwrap()
+}
+
+async fn clear(client: &Client, url: &str) {
+    let res = client
+        .post(url)
+        .header("X-Op", "clear")
+        .header(reqwest::header::CONTENT_TYPE, "application/octet")
+        .send()
+        .await
+        .unwrap();
+    assert!(res.status().is_success());
+}
+async fn num(client: &Client, url: &str) -> u64 {
+    let res = client
+        .post(url)
+        .header("X-Op", "num")
+        .header(reqwest::header::CONTENT_TYPE, "application/octet")
+        .send()
+        .await
+        .unwrap();
+    assert!(res.status().is_success());
+    let num_peers: u64 =
+        kitsune_p2p_types::codec::rmp_decode(&mut res.bytes().await.unwrap().as_ref()).unwrap();
+    println!("num_peers {}", num_peers);
+    num_peers
+}
+async fn sync(client: &Client, num_wait_for: u64, url: &str) {
+    let mut body_data = Vec::new();
+    kitsune_p2p_types::codec::rmp_encode(&mut body_data, num_wait_for).unwrap();
+    let res = client
+        .post(url)
+        .header("X-Op", "sync")
```

Comment:
I can't find this operation on the bootstrap server side. is there an implied default behavior that's used here?

Reply:
Resolving this myself as I found that the counterpart comes with this PR and I was looking on the `develop` branch :wink: